.github/workflows/maven.yml

```
@@ -30,4 +30,6 @@ jobs:
         java-version: '11'
         distribution: 'adopt'
     - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      run: mvn -B install --file pom.xml
+    - name: Test deployment
+      run: mvn -B -pl main-ui -am install -Ptest-deployment --file pom.xml
```

Is `--file pom.xml` really needed? Shouldn't it be the default?
Also `-am` should not be needed since we installed all modules in the previous step
```suggestion
      run: mvn -B -pl main-ui install -Ptest-deployment
```